database/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM postgres:9.6.2-alpine
+
+RUN apk update \
+&& apk upgrade \
+&& apk add -q alpine-sdk \
+&& apk add -q vim \
+&& apk add -q nano;
+
```

Comment:
In the Dockerfile we will not. At this point there is no database server running.

But you can place script sin /docker-entrypoint-initdb.d/ and those will be run when the database is started.